degoogle.yml

```diff
@@ -1318,4 +1323,4 @@ hardware:
     - title: FitBit
     - notes:
       - |
-        Due to Google's  [recent acquisition of FitBit](https://outline.com/tsJCYN), it is recommended to avoid using FitBit. Instead, check out this [list of smart watches](https://asteroidos.org/install/) supported by Asteroid OS (See Wear OS in [Operating systems](#operating-systems)). Thanks @ribtoks
\ No newline at end of file
+        Due to Google's  [recent acquisition of FitBit](https://outline.com/tsJCYN), it is recommended to avoid using FitBit. Instead, check out this [list of smart watches](https://asteroidos.org/install/) supported by Asteroid OS (See Wear OS in [Operating systems](#operating-systems)). Thanks @ribtoks
```

Comment:
What is actually different here? Or is GitHub just acting up again?

Reply:
Update: I did some checksums on each line and they're identical. Looks like GitHub is acting up again :)